app/models/operatingsystems/vsphere.rb

```diff
@@ -0,0 +1,23 @@
+class Vsphere < Operatingsystem
+  PXEFILES = {:kernel => "mboot.c32", :initrd => ""}
+
+  class << self
+    delegate :model_name, :to => :superclass
+  end
+
+  def mediumpath(host)
+    medium_uri(host).to_s
+  end
+
+  def pxe_type
```

Comment:
This needs also 

```ruby
def pxedir
  "EFI/BOOT"
end
```

Not sure 100% about whether that path suits all versions but for 6.0 that seems OK